### DISCUSSION TITLE
视频内显示标题

### DISCUSSION THREAD

**Discussion by Kinda2419** [the1812/Bilibili-Evolved#4221] [功能建议 / Ideas]
我在各个页面使用"win+k"查找后均未发现有"视频内显示标题"的相关问题,请问是否已经提供了"视频内显示标题"的功能?如果有,还请指个路,谢谢.

**Comment by Geeyun-JY3** [comment]
网页全屏和全屏时破站本身就自带一个标题层啊。
还是说你想要任意时候都有一个标题层。

**Reply by Geeyun-JY3** [reply]
#3831 提到了个脚本：https://greasyfork.org/zh-CN/scripts/440820
